test/browser-headers.test.ts

```diff
@@ -1,18 +1,21 @@
-const http = require('http');
-const HeaderGenerator = require('header-generator');
-const { got } = require('got-cjs');
+import { URL } from 'url';
+import { AddressInfo } from 'net';
+import http, { Agent, Server } from 'http';
+// @ts-expect-error Missing types
+import HeaderGenerator from 'header-generator';
+import { got, Options } from 'got-cjs';
 
-const { browserHeadersHook, mergeHeaders } = require('../src/hooks/browser-headers');
-const TransformHeadersAgent = require('../src/agent/transform-headers-agent');
-const gotScraping = require('../src/index');
+import { browserHeadersHook, mergeHeaders } from '../src/hooks/browser-headers';
+import { TransformHeadersAgent } from '../src/agent/transform-headers-agent';
+import gotScraping, { OptionsInit } from '../src/index';
 
-const { startDummyServer } = require('./helpers/dummy-server');
+import { startDummyServer } from './helpers/dummy-server';
 
 describe('Browser headers', () => {
-    let options;
-    let generatorSpy;
-    let server;
-    let port;
+    let options: OptionsInit;
```

Comment:
seems like if you type this as just `Options` you will safe yourself from a lot of type casting